### ISSUE TITLE
The orentation of the sockets should be turned by 180°

### ISSUE THREAD

**Issue by KarlZeilhofer** [prusa3d/MM-control-2.0#2]
Is there any chance, that the sockets will be turned by 180°?  
With the current case of the MMU 2.0 is nearly impossible to unplug the cables when everything is assembled.

**Comment by ok1hra** [comment]
We do not have the same feeling.

**Comment by KarlZeilhofer** [comment]
No, why not? I think to remember, that the trouble shooting guide says to use an allen key to unlock the securing pin of the communication plug.

**Comment by domesticatedviking** [comment]
I very much agree.  When using an enclosure, all of the connections to the MMU2 are in a location that can neither be seen nor reached from the front of the printer, and they all require tools to remove.   The result is that it is VERY difficult to remove the printer for maintenance.   This is why I designed the [underslung control panel mod](https://www.thingiverse.com/thing:3211711).
